configure.py

```diff
@@ -36,9 +38,9 @@ def update_cache(self, update_single=None):
                     continue
             wokwi_id = self.install_local_artifacts(number, url)
             if wokwi_id in self.wokwi_ids:
-                logging.error("wokwi id already exists!")
-                exit(1)
-            self.wokwi_ids.append(wokwi_id)
+                logging.warning(f"wokwi id {wokwi_id} already exists!")
```

Comment:
what is the idea behind continuing if there is a duplicate? I'd say this is an error worth stopping for.

Reply:
I'll admit, this was just a bit of a hack to get around problems with assembling the designs